errors/sync/isPositionChoiceError.js

```diff
@@ -2,14 +2,11 @@ const Validator = require('is-my-json-valid')
 const schema = require('../schema/positionChoiceError')
 const validator = Validator(schema, {verbose: true})
 
-// server is not used here. Closure pattern is just for consistency of use with other functions.
-module.exports = function (server) {
-  return function isError (obj) {
-    const result = validator(obj)
+module.exports = function isError (obj) {
+  const result = validator(obj)
```

Comment:
maybe call this function isPositionChoiceError?

Reply:
done